Library/Homebrew/formula.rb

```diff
@@ -475,8 +473,9 @@ def inspect
   # less consistent and the standard parameters are more memorable.
   def std_cmake_args
     %W[
+      -DCMAKE_C_FLAGS_RELEASE=""
```

Comment:
This is fine, actually, let's do this and the same for -DCMAKE_CXX_FLAGS_RELEASE.